src/Authentication.java

```diff
@@ -1,14 +1,17 @@
 public class Authentication {
 
-    public static void validation(String login, String password, String confirmPassword)
-            throws WrongLoginException, WrongPasswordException {
+    public static void validation(String login, String password, String confirmPassword) {
 
-        Checker.checkLoginLength(login);
-        Checker.checkLoginCharacters(login);
+        try {
+            Checker.checkLoginLength(login);
+            Checker.checkLoginCharacters(login);
 
-        Checker.checkPasswordLength(password);
-        Checker.checkPasswordCharacters(password);
-        Checker.checkConfirmPassword(password, confirmPassword);
+            Checker.checkPasswordLength(password);
+            Checker.checkPasswordCharacters(password);
+            Checker.checkConfirmPassword(password, confirmPassword);
+        } catch (WrongLoginException | WrongPasswordException e) {
+            throw new RuntimeException(e);
```

Comment:
не нужно пробрасывать исключение выше по стеку, надо выводить на печать сообщение - это и будет обработка
```suggestion
            System.out.println(e.getMessage());
```